src/test/java/nixy/parser/ParserTest.java

```diff
@@ -0,0 +1,133 @@
+package nixy.parse;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import nixy.Command;
+import nixy.exceptions.NixyException;
+import nixy.task.DeadlineTask;
+import nixy.task.EventTask;
+import nixy.task.TodoTask;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ParserTest {
+
+    @Test
+    void byeCommand() {
+        Parsed result = Parser.parse("bye");
+        assertEquals(Command.BYE, result.getCommand());
+    }
+
+    @Test
+    void listCommand() {
+        Parsed result = Parser.parse("list");
+        assertEquals(Command.LIST, result.getCommand());
+    }
+
+    @Test
+    void parseMarkAsDone() {
+        Parsed result = Parser.parse("mark 1");
+        assertEquals(Command.MARK, result.getCommand());
+        assertEquals(1, result.getTaskNumber());
+    }
+
+    @Test
+    void markAsDoneInvalidNumberFormat() {
```

Comment:
Could this be renamed as `markAsDone_InvalidNumberFormat()`? I noticed the same issue in some of the below tests as well. Test method names typically follow this format: `featureUnderTest_testScenario_expectedBehavior()`.

Reply:
great catch! I had missed the relevant information regarding test methods naming conventions